pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
fwiw `frontend-service-make` respects `#` comments in `deploy-whitelist.txt`. 

Though I guess this is fine -- it should be documented in the readme that this sorting hook **will** remove blank lines and **won't** respect comments (unlike `requirements-txt-sorter`)